x/bank/src/keeper/mod.rs

```diff
@@ -597,7 +353,7 @@ impl<SK: StoreKey, PSK: ParamsSubspaceKey, AK: AuthKeeper<SK, M>, M: Module>
         ctx: &mut CTX,
         coin: UnsignedCoin,
     ) -> Result<(), GasStoreErrors> {
-        // TODO: need to delete coins with zero balance
+        // If there will be any issue with bank then delete coins with zero balance. See for more details: https://github.com/NYBACHOK/gears/blob/c79c50f60e40cee3fbc050002bf1ce63a72b67a3/x/bank/src/keeper/bank.rs#L70-L72
```

Comment:
There is an issue. Coins with zero balance need to be deleted.

Reply:
And you do this, but on the other level.